doc/book/helpers/escape.md

```diff
@@ -0,0 +1,248 @@
+# Escape
+
+The following helpers can **escape output in view scripts and defend from XSS
+and related vulnerabilities**. To escape different contexts of a HTML document,
+zend-view provides 5 helpers:
```

Comment:
I don't like fixed numbers, it's a very big chance that it will be not update if the number of escapers change.
Maybe better "provides the following helpers:" ?

Reply:
Very good point! 👍